diff --git a/cjs/react-server-dom-webpack-server.browser.development.js b/cjs/react-server-dom-webpack-server.browser.development.js
index 3e348765670b75a2d2aeffa91e4532b0dc52394f..c744cbf613d37dcafe3569fcd63ab5c7b4655b53 100644
--- a/cjs/react-server-dom-webpack-server.browser.development.js
+++ b/cjs/react-server-dom-webpack-server.browser.development.js
@@ -355,7 +355,7 @@
                 });
                 break;
               case "stylesheet":
-                preload(srcSet, "stylesheet", {
+                preload(srcSet, "style", {
                   crossOrigin: props.crossOrigin,
                   integrity: props.integrity,
                   nonce: props.nonce,
diff --git a/cjs/react-server-dom-webpack-server.browser.production.js b/cjs/react-server-dom-webpack-server.browser.production.js
index 7c930da835b6abeb26203f664cf94683eb9b4af5..353c6fc8c45db48c5f3a31fe819c43e109c01185 100644
--- a/cjs/react-server-dom-webpack-server.browser.production.js
+++ b/cjs/react-server-dom-webpack-server.browser.production.js
@@ -466,7 +466,7 @@ function getChildFormatContext(parentContext, type, props) {
             });
             break;
           case "stylesheet":
-            preload(srcSet, "stylesheet", {
+            preload(srcSet, "style", {
               crossOrigin: props.crossOrigin,
               integrity: props.integrity,
               nonce: props.nonce,
diff --git a/cjs/react-server-dom-webpack-server.edge.development.js b/cjs/react-server-dom-webpack-server.edge.development.js
index 31e0f9bc64031eded1ffc128d624b6b520bcccf9..9329125d1a6ea83a057b0b2a0dd8e4c2c0d1bb7c 100644
--- a/cjs/react-server-dom-webpack-server.edge.development.js
+++ b/cjs/react-server-dom-webpack-server.edge.development.js
@@ -359,7 +359,7 @@
                 });
                 break;
               case "stylesheet":
-                preload(srcSet, "stylesheet", {
+                preload(srcSet, "style", {
                   crossOrigin: props.crossOrigin,
                   integrity: props.integrity,
                   nonce: props.nonce,
diff --git a/cjs/react-server-dom-webpack-server.edge.production.js b/cjs/react-server-dom-webpack-server.edge.production.js
index fffccc52f0668287f11d0a8b7a9fc4aae5fc3cee..41659aff52832c5828ef813f9a7018da71933e77 100644
--- a/cjs/react-server-dom-webpack-server.edge.production.js
+++ b/cjs/react-server-dom-webpack-server.edge.production.js
@@ -476,7 +476,7 @@ function getChildFormatContext(parentContext, type, props) {
             });
             break;
           case "stylesheet":
-            preload(srcSet, "stylesheet", {
+            preload(srcSet, "style", {
               crossOrigin: props.crossOrigin,
               integrity: props.integrity,
               nonce: props.nonce,
diff --git a/cjs/react-server-dom-webpack-server.node.development.js b/cjs/react-server-dom-webpack-server.node.development.js
index 5e6c396af15955c5e4a5ba23c24ef6f25531ce7b..3a1ccb280663c6a7d50fcfbe51f78854f6946f75 100644
--- a/cjs/react-server-dom-webpack-server.node.development.js
+++ b/cjs/react-server-dom-webpack-server.node.development.js
@@ -388,7 +388,7 @@
                 });
                 break;
               case "stylesheet":
-                preload(srcSet, "stylesheet", {
+                preload(srcSet, "style", {
                   crossOrigin: props.crossOrigin,
                   integrity: props.integrity,
                   nonce: props.nonce,
diff --git a/cjs/react-server-dom-webpack-server.node.production.js b/cjs/react-server-dom-webpack-server.node.production.js
index 93aea0a1007874afea1da8606b30f950bf4fab0e..e8eedfd9acfce04630fa616816185d539460f7a2 100644
--- a/cjs/react-server-dom-webpack-server.node.production.js
+++ b/cjs/react-server-dom-webpack-server.node.production.js
@@ -503,7 +503,7 @@ function getChildFormatContext(parentContext, type, props) {
             });
             break;
           case "stylesheet":
-            preload(srcSet, "stylesheet", {
+            preload(srcSet, "style", {
               crossOrigin: props.crossOrigin,
               integrity: props.integrity,
               nonce: props.nonce,
